lib/mobius/cog_utils.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Mobius.CogUtils do
+  @moduledoc """
+  A module of utilities for cogs such as formatting tables
+  """
+
+  @type categories :: [{String.t(), [{String.t(), String.t()}]}]
+
+  @doc ~S"""
+  Formats a list of tuples such that the spacing aligns the 2nd element of the inner tuple
+
+  ## Example
+
+      iex> data = [{"Cat1", [{"abc", "description"}, {"defghi", "something"}]}]
+      iex> format_categories_list(data)
+      "Cat1:\n  abc       description\n  defghi    something"
```

Comment:
```suggestion
      ~S"
      Cat1:
        abc       description
        defghi    something
      "
      |> String.trim()
```
What do you think of that? It's definitely more verbose, but I think it's much more clear as to how the input gets formatted exactly.

Reply:
It messes with the spacing on the 2nd and 3rd lines though. I don't think there's a quick and easy way to make it work. Also, is the expected value executed? I thought it had to be matching what's printed or something like that 🤔 I'll try to make it better, but I'm a bit skeptical

Reply:
Did you test it? It works as expected for me.

![image](https://user-images.githubusercontent.com/44734484/119362816-1ee11980-bc7b-11eb-8f24-3400cecad773.png)
![image](https://user-images.githubusercontent.com/44734484/119362914-3ae4bb00-bc7b-11eb-8231-5693ce9440fc.png)
![image](https://user-images.githubusercontent.com/44734484/119363002-551e9900-bc7b-11eb-8f5a-284b62d3e27c.png)

Reply:
I'm so confused by how this works. I'm expecting 6 spaces before "Cat1" in your screenshots and I don't get why that's not the case. I agree it's much better, I didn't know it worked like that. Thank you!

Reply:
I guess the doctest algorithm accounts for that and "aligns" the sigil with the `iex>` prompt 🤷